docs/api-reference/uiSchema.md

```diff
@@ -64,7 +109,33 @@ Will result in:
 </div>
 ```
 
-## autofocus
+#### BREAKING CHANGE in version 5
+In versions previous to 5, `classNames` was the only property that did not require the `ui:` prefix. 
+Additionally, it did not support being added into the `ui:options` object.
+This was fixed in version 5 to be consistent with all the other properties in this section.
+If you were using `classNames` as follows, simply add the `ui:` prefix to it:
+
+```jsx
+const uiSchema = {
+  title: {
+    "classNames": "my class"
```

Comment:
For clarity, we should provide an example with `ui:classNames` in it

Reply:
If you look at the top of the file you'll see that I do provide `ui:classNames` in the main example of equivalence. Is that sufficient?